cmd/runner/runner.go

```diff
@@ -206,12 +206,15 @@ func createCommands(
 		return []*command{}, nil, fmt.Errorf("Cannot create go builder: %w", err)
 	}
 
-	// Rebuild whenever a go file changes
+	// Rebuild whenever an application file changes
 	compilerDebouncer := debounce.New(time.Duration(cfg.Tools.Debounce) * time.Millisecond)
 	go onEvent(evFileChanged, func(ev event) bool {
-		if strings.HasSuffix(ev.payload, ".go") {
+		if strings.HasSuffix(ev.payload, ".go") || // source file
+			strings.HasSuffix(ev.payload, "/go.sum") || // dependency
+			strings.HasSuffix(ev.payload, "/config.toml") || // configuration
+			strings.Contains(ev.payload, "/.env") { // environment configuration
```

Comment:
Dit zou dan ook getriggerd worden door een aanpassing aan `.env.example`, mss geen groot probleem, maar waarom gebruik je hier `Contains` ipv `HasSuffix`?

Reply:
Omdat `.env` heel vaak met verschillende environments gebruikt wordt, bv. `.env.production`, `.env.staging`, `.env.local`. Ik wil die allemaal matchen zodat we altijd gaan reloaden en niet hoeven te weten welke environment je aan het uitvoeren bent (veel mensen gebruiken bv. ook enkel `.env.local` en geen `.env` zonder meer). Contains doet een reload voor al die gevallen. Dat gaat inderdaad ook onnodig herladen als je `.env.example` updatet maar dat maakt op zich niet echt uit, dat bestand verandert bijna nooit 😛